### DISCUSSION TITLE
how to pass argv to compile option?

### DISCUSSION THREAD

**Discussion by WingDust** [nvim-treesitter/nvim-treesitter#4229] [Q&A]
if i use `clang` for `TSInstallFromGrammar` ,
how to pass argv to compile option? 
like  `clang: error: linker command failed with exit code 1120 (use -v to see invocation)`  explain by pass argv `-v`

**Comment by theHamsta** [comment]
In the readme, there is an explanation on how to add extra options to curl. The same trick works for `cc` if clang was invoked through the cc command (you can see in `:checkhealth nvim-treesitter`. Additional compile flags can also be added when editing our source code in select_shell_commands.lua
